Patches/Harpy/Patch_Prosthetics_Harpy.xml

```diff
@@ -0,0 +1,158 @@
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>steel harpy claw</label>
+							<capacities>
+								<li>Cut</li>
+							</capacities>
+							<power>14</power>
+							<cooldownTime>0.87</cooldownTime>
+							<armorPenetrationSharp>3</armorPenetrationSharp>
+							<armorPenetrationBlunt>2.5</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>steel harpy claw</label>
+							<capacities>
+								<li>Stab</li>
+							</capacities>
+							<power>12</power>
+							<cooldownTime>0.87</cooldownTime>
+							<armorPenetrationSharp>5</armorPenetrationSharp>
+							<armorPenetrationBlunt>2.5</armorPenetrationBlunt>
+						</li>
+					</tools>
+				</value>
+			</li>
+			
+			<!-- Plasteel Claw -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="PlasteelHarpyClaw"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>plasteel harpy claw</label>
+							<capacities>
+								<li>Cut</li>
+							</capacities>
+							<power>18</power>
+							<cooldownTime>0.93</cooldownTime>
+							<armorPenetrationSharp>5.5</armorPenetrationSharp>
+							<armorPenetrationBlunt>3</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>plasteel harpy claw</label>
+							<capacities>
+								<li>Stab</li>
+							</capacities>
+							<power>12</power>
+							<cooldownTime>0.93</cooldownTime>
+							<armorPenetrationSharp>7.5</armorPenetrationSharp>
+							<armorPenetrationBlunt>3</armorPenetrationBlunt>
+						</li>
+					</tools>
+				</value>
+			</li>
+			
+			<!-- Mono Claw -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="HarpyMonoClaw"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>harpy monoclaw</label>
+							<capacities>
+								<li>Cut</li>
+							</capacities>
+							<power>26</power>
+							<cooldownTime>0.78</cooldownTime>
+							<armorPenetrationSharp>12</armorPenetrationSharp>
+							<armorPenetrationBlunt>8</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>harpy monoclaw</label>
+							<capacities>
+								<li>Stab</li>
+							</capacities>
+							<power>14</power>
+							<cooldownTime>0.78</cooldownTime>
+							<armorPenetrationSharp>16</armorPenetrationSharp>
+							<armorPenetrationBlunt>8</armorPenetrationBlunt>
+						</li>
+					</tools>
+				</value>
+			</li>
+			
+			<!-- Venom Claw -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="HarpyVenomClaw"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>harpy venonmclaw</label>
+							<capacities>
+								<li>ScratchToxic</li>
+							</capacities>

```

Comment:
I'd reduce this to around 14 or so, as the poison strength is related to the power of the attack that inflicts it. It's going to be quite powerful at this damage amount. Also, why does this have two attacks for ScratchToxic?